### PR TITLE
Added a local development server to preview.

### PR DESCRIPTION
**Pull request by Janiceilene**
### Description
I created a local development server so everyone can preview the website.

Fixes # [ISSUE]

### Type of Change:
**Delete irrelevant options.**

- Code
- Documentation
  

**Code/Quality Assurance Only**
- This change requires a documentation update (software upgrade on readme file)
- New feature (non-breaking change which adds functionality pre-approved by mentors)

### How Has This Been Tested?
Describe the tests you ran to verify your changes. Provide instructions or GIFs so we can reproduce. List any relevant details for your test.

### Checklist:
**Delete irrelevant options.**

- [x] My PR follows the style guidelines of this project
- [x] I have performed a self-review of my own code or materials
- [ ] I have commented my code or provided relevant documentation, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation

I'm going to fix documentation after pushing this PR
  

**Code/Quality Assurance Only**
- [x] My changes generate no new warnings

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+  "scripts": {
+    "start": "ecstatic --port=8050 .",
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/systers/systers.github.io.git"
+  },
+  "author": "",
+  "license": "UNLICENSED",
```
Comment (sara-02) [inline package.json]: @Janiceilene I have a doubt here, in our project the LICENSE file points to  GNU GPL License, https://github.com/systers/systers.github.io/blob/master/LICENSE
So are we good with using the `UNLICENSED` in the node module?

File: .gitignore [hunk 2]
```diff
@@ -1 +1,2 @@
-.DS_Store
\ No newline at end of file
+.DS_Store
+node_modules
```
Comment (simsausaurabh) [inline .gitignore]: New line at the end of file needs to be inserted.

File: package.json (hunk 1 above)
Comment (Janiceilene) [reply package.json]: The GNU GPL license isn't necessarily correct.  I need to talk with @MeepyMay and figure out what license should be used for the project, but she's out of town and I wanted to push this asap, so I went with unlicensed for now.

Comment (MeepyMay) [reply package.json]: Yeah, we didn't decide. Can you update on the next PR? I don't think we'll have the bandwidth to research all of the options. So we can add the GPU 2.0 for all repos, except the crowdmap/mailman3-related repos. They have MIT's license.

**Pull request merged**